docs/specs/servers/blockchain/blockchain-bundler-api.md

```diff
@@ -0,0 +1,51 @@
+# Blockchain API Bundlers interaction
+
+This API provides a bundlers JSON-RPC interaction. 
+The supported `userOperation` interaction schema is according to v0.7 ERC-4337 specification.
+
+## Make a bundler JSON-RPC request
+
+Used to make a JSON-RPC call to the designated bundler.
+
+`POST /v1/bundler?projectId={projectId}&chainId={chainId}&bundler={bundlerName}`
+
+* `projectId` - Required. The project identifier.
+* `chainId` - CAIP-2 chain identifier.
```

Comment:
for eip5792 methods, I'd like to be able to use it without chainId, is it possible?

Reply:
Ok, let’s use chainId from session request